app/assets/stylesheets/components/dashboard/profile_card.scss

```diff
@@ -29,3 +29,9 @@
     color: $grey;
   }
 }
+
+.profile-card h2 {
```

Comment:
Is this needed? I'd be reluctant to add anything to this file because it will all be [deleted soon](https://github.com/TheOdinProject/theodinproject/pull/3381)

Reply:
Hello! This code will remove the padding necessary to make the username centered in the profile card when the window is resized! I was not sure in which file I should put this code, so maybe you know a better place than me! What's important is to find a way to tag that h2 element and remove the padding on breakdown.

I attached two pictures to further show you the padding in both [maximized window](https://user-images.githubusercontent.com/112586034/194549173-9746ac20-f8d4-4ec7-a5ab-382b0c233d9f.png) and [resized window](https://user-images.githubusercontent.com/112586034/194549167-9ebbc602-1ae0-4819-ad3d-b8f14e5153ef.png). As you can see when you resize the window enough, the panel changes, and that padding is no longer useful, in fact, its making the username not centered.

Reply:
Gotcha, thanks for explaining.

We would ideally fix this using Tailwind, but it won't be straightforward as there is a mismatch between the breakpoints we use in the css in sass and what Tailwind uses. 

I think we should hold fire with this until https://github.com/TheOdinProject/theodinproject/pull/3381 is merged , this issue will be pretty easy to fix once those changes are in place.

Reply:
Dashboard has been changed over, we're good to proceed with fixing this using Tailwind instead @rtins.